scripts/populate_panels/load_g2p_data.pl

```diff
@@ -990,3 +998,88 @@ sub add_public_comments {
   return $count;
 
 }
+
+sub add_ontology_accession {
+   my ($disease, $disease_mim, $disease_mondo) = @_;
+   my @mondo = get_ontology_accession($disease_mim, $disease_mondo);
+   if (scalar @mondo > 0){
+     my $attribute = "Data source";
+     my $disease_id = $disease->dbID;
+     my $mapped_by_attrib = $attrib_adaptor->get_attrib('ontology_mapping', $attribute);
+     foreach my $mondo (@mondo){
+       my $ontology_accession_id = $mondo->ontology_term_id;
+       my $dom = Bio::EnsEMBL::G2P::DiseaseOntology->new(
+        -disease_id => $disease_id,
+        -ontology_term_id => $ontology_accession_id,
+        -mapped_by_attrib => $mapped_by_attrib,
+        -adaptor => $disease_ontology_adaptor,
+       );
+       $dom = $disease_ontology_adaptor->store($dom);
+     }
+   }
+  
+  print $fh_report "Disease ontology mapping has been added to the database"; 
+ 
+}
+sub check_ontology_accession {
+  my @list = shift;
+  my $ontology_accession = join (',', @list);
+  print STDERR "    ADD: Planning to add ontologies $ontology_accession\n";
+ 	  
+}
+sub get_ontology_accession {
+  my ($disease_mim, $disease_mondo) = @_;
+  my $mondo_description = "OLS extract";
+  my $given_mondo_descript = "Term by Curator";
+  my @mondos_stored;
+  my @mondos_only;
+  if (defined($disease_mim) && !defined($disease_mondo)){
+    my $server = 'http://www.ebi.ac.uk/ols/api/search?q=';
+    my $ontology = '&ontology=mondo';
+    my $request = $server . $disease_mim . $ontology;
+    my $response = $http->get($request, 
+             {headers => { 'Content-type' => 'application/xml' }
+    });
+    warn "Failed!\n" unless $response->{success};
+    my $result = JSON->new->decode($response->{content});
+    foreach my $id  (@{$result->{response}->{docs}}){
+      if ($id->{obo_id}  =~   m/MONDO/){
+        push @mondos_only, $id->{obo_id}; 
+      }
+    }
+    
+    foreach my $mondo_id (@mondos_only){
+      my $mondos = $ontology_accession_adaptor->fetch_by_accession($mondo_id);
+      if (! defined $mondos){
+        my $mondo = Bio::EnsEMBL::G2P::OntologyTerm->new(
+          -ontology_accession => $mondo_id,
+          -description        => $mondo_description,
+          -adaptor            => $ontology_accession_adaptor,
+        );
+        $mondo = $ontology_accession_adaptor->store($mondo);
+        push @mondos_stored, $mondo;
+        
+
+      }
+    }
+    
+  }
+
+  if ($disease_mondo && !defined($disease_mim) ){	  
+    $disease_mondo =~ m/MONDO/;
+    my $mondos = $ontology_accession_adaptor->fetch_by_accession($disease_mondo);
+    if (!defined($mondos)){
+      my $mondo = Bio::EnsEMBL::G2P::OntologyTerm->new(
+          -ontology_accession => $disease_mondo,
+          -description        => $given_mondo_descript,
```

Comment:
The description we would want to store here is the MONDO term. So if the accession submitted is MONDO:0016587 the description would be arrhythmogenic right ventricular cardiomyopathy. The same goes for the above.

We can leave this null in the first instance to get data in.